network-api/networkapi/buyersguide/templates/product_page.html

```diff
@@ -38,11 +38,11 @@
       <div class="row intro d-flex justify-content-between align-items-center">
         <a id="product-company-url" class="product-price small-link col-md-8 mb-3" href="{{product.url}}" target="_blank">{{product.company}} ${{product.price}}</a>
         <div class="copy-link-wrapper col-md-4 mb-3 text-md-right">
-          <a id="product-copy-link-button" class="copy-link small-link" href="#" data-url={{product.url}} data-success-text="link copied!">copy & share link</a>
+          <a id="product-copy-link-button" class="copy-link small-link" href="#" data-url={{product.url}} data-success-text="{% trans 'link copied!' %}">{% trans "copy & share link" %}</a>
         </div>
         <h2 class="h1-heading col-md-10">{{product.name}}</h2>
         <div class="col-12">
-          <p class="body-small">Review Date {{product.review_date|date:"m/d/y"}}</p>
+          <p class="body-small">{% blocktrans with date=product.review_date|date:"SHORT_DATE_FORMAT" context "Short date format (e.g. 12/31/2003)" %}Review Date {{ date }}{% endblocktrans %}</p>
```

Comment:
nit, but: let's stick with sane date formatting in the context hint, because the US uses a ridiculous format, and everyone else uses the imminently sensible "longer-to-shorter" format. so I'm 100% up for using 2003/12/31 in the context hint.

Reply:
Fixed!